### ISSUE TITLE
Not transfer/received voice after cellular call ended

### ISSUE THREAD

**Issue by nfasate** [twilio/video-quickstart-swift#152]
If I am on a Twilio video call and at the same time I received a cellular call on my device, I accept the call, speak with the person and end the call, I resume on Twilio video call everything is working apart from my voice is not getting transfer to the other end, also other end’s voice is not received at my end.

Currently, I am using 'TwilioVideo', '~> 1.2.1' SDK's.

**Comment by GuntisTreulands** [comment]
Hi @nfasate,

Couple of questions:

1.) Did you use CallKit in your Twilio call?
2.) Was the cellular call outgoing, or incoming?
3.) On what iOS versions/devices you encountered this issue?

Guntis.

**Comment by nfasate** [comment]
Hello Guntis,

Thanks for the quick reply.

1.) Did you use CallKit in your Twilio call?
- No, We used open URL.
  `UIApplication.shared.open(URL(string: "tel://\(phoneNumber)"))`

2.) Was the cellular call outgoing, or incoming?
- We are facing the issue in both outgoing and incoming.

3.) On what iOS versions/devices you encountered this issue?
- iOS - 9 and above / 5S, 6+, 5C.

Thanks,
Nilesh

**Comment by GuntisTreulands** [comment]
@nfasate 

But you are experiencing this issue with  
VideoCallKitQuickStart.xcodeproj  or with VideoQuickStart.xcodeproj
or both?

(If you are using your demo application, can you try to reproduce this issue on one of those demo applications? )


I could not reproduce that issue on VideoQuickStart.xcodeproj (Tried with iOS 9.3.1, iOS 10.3.3, iOS 11  outgoing and incoming calls). 

But I got a possible issue on VideoCallKitQuickStart.xcodeproj (iOS 10+ only, because it uses CallKit). 
When I have an active Twilio call between two devices, then one of the devices makes outgoing cellular call (by going to Phone app, or opening URL to call a number), and after 5-10 seconds (active cellular call), that cellular call ends and Twilio app audio and/or video appears not working (no audio, or video appears black). 

Could it be that this described issue is also your case, or is your case a bit different?


Guntis.

**Comment by nfasate** [comment]
Hi Gantis,

> But you are experiencing this issue with
VideoCallKitQuickStart.xcodeproj or with VideoQuickStart.xcodeproj
or both?

I am experiencing this issue with VideoQuickStart.xcodeproj.

> (If you are using your demo application, can you try to reproduce this issue on one of those demo applications? )

Yes, I will reproduce the scenario on the same demo and let you know the behavior. 

> When I have an active Twilio call between two devices, then one of the devices makes outgoing cellular call (by going to Phone app, or opening URL to call a number), and after 5-10 seconds (active cellular call), that cellular call ends and Twilio app audio and/or video appears not working (no audio or video appears black).

Yes, this actual scenario I am facing on VideoQuickStart.xcodeproj.

**Comment by ptankTwilio** [comment]
@nfasate Thanks for the information. Unfortunately, we are not able to reproduce the problem. Did you make any code changes in QuickStart ? If yes, Is it possible for you to share the changes to help reproduce the problem?

**Comment by nfasate** [comment]
@ptankTwilio  Yes, we have done the required changes in QuickStart Demo and sharing the source link below.
You can download the source code without login. 

https://www.dropbox.com/s/hkaloehm5eozaer/QuickStart%20Demo.zip?dl=0

Thanks

**Comment by ptankTwilio** [comment]
@nfasate Thanks for making the required changes in QuickStart Demo and sharing the code with us.   We will post an update soon on this.

**Comment by pravinsonicjobs** [comment]
Is this issue resolved as I am also facing same problem without making changes in demo project?
